ansible/modules/hashivault/hashivault_azure_auth_role.py

```diff
@@ -133,9 +133,9 @@ def main():
     argspec['num_uses'] = dict(required=False, type='int', default=0)
 
     supports_check_mode=True
```

Comment:
Should this really be gone?

Reply:
so for some reason, python 36 and 27 fail on this line... something about "too many arguments". however, 37 it works... i removed it but left it for future re-integration. i probably should have included a note on that though